training/provision/playbook.yml

```diff
@@ -37,7 +72,9 @@
       privileged: yes
       pid_mode: host
       command: "bootc install to-filesystem --karg=console=ttyS0,115200n8 --replace=alongside /target"
+    become: true
```

Comment:
I don't believe this is necessary as there's a toplevel `become: true` on the whole playbook.